google/cloud/bigtable/examples/README.md

```diff
@@ -70,12 +82,57 @@ offers a simple way to do so:
 $ gcloud auth login
 ```
 
-## Running the Quick Start Examples using a Cloud Bigtable Instance
+## Samples
+
+### Hello world
+
+View the [Hello World][hello_world_code] sample to see a basic usage of
```

Comment:
View the [....] example to see sample usage of the Cloud Bigtable client library.

Reply:
fixed

Reply:
PTAL